common/app/routes/challenges/redux/reducer.js

```diff
@@ -81,7 +82,13 @@ const mainReducer = handleActions(
     }),
     [types.updateTests]: (state, { payload: tests }) => ({
       ...state,
-      tests
+      tests,
+      isChallengePassed: (tests.length > 0 &&
```

Comment:
I need to use this prop in the store to open and close the modal. If the camper closes the modal to stay on the current challenge, how can I set this to false if it is coming from the selector.

The selector with always set this to the what the conditional resolves as. 

Can I move the conditional to the action creator for `updateTests`?